src/main/java/reposense/util/FileUtil.java

```diff
@@ -38,6 +41,52 @@
     private static final String GITHUB_API_DATE_FORMAT = "yyyy-MM-dd";
     private static final ByteBuffer buffer = ByteBuffer.allocate(1 << 11); // 2KB
 
+    /**
+     * Returns a list of relevant repo paths from {@code repoConfigsList}.
+     */
+    public static Set<Path> getReportFolders(Path sourcePath, List<RepoConfiguration> repoConfigsList) {
+        Set<Path> relevantFolders = new HashSet<>();
+        for (RepoConfiguration repoConfiguration : repoConfigsList) {
+            relevantFolders.add(Paths.get(sourcePath + File.separator + repoConfiguration.getDisplayName())
+                    .toAbsolutePath());
+        }
+        return relevantFolders;
```

Comment:
No duplicates. I used set due to the previous implementation of the code that will impede performance if list is used instead of set, which has already been re-implemented.

Have changed it to use list instead.